### ISSUE TITLE
Latest reading installer, not version installed for Obsidian app

### ISSUE THREAD

**Issue by porcherie** [mangerlahn/Latest#359]
<img width="1277" alt="SCR-20240227-pood" src="https://github.com/mangerlahn/Latest/assets/50797800/9a142c55-3327-4eac-8b47-149289a464f7">

Latest should not show Obsidian as update as most recent version is already installed

**Comment by porcherie** [comment]
Ps it just came to my mind that it would be cool to have the functionality to skip an update, kind like the ignore functionality but temporary

**Comment by luckman212** [comment]
For Obsidian, since it's an Electron app, it uses .asar patch files to do incremental updates. Would be difficult (impossible?) for Latest to read these.

If an update is indicated, it typically indicates a new "installer" version available which also updates the bundled Chrome engine etc. So it's always good to download the updated .dmg anyway even if you "already" have the latest version installed.

**Comment by hortonew** [comment]
> For Obsidian, since it's an Electron app, it uses .asar patch files to do incremental updates. Would be difficult (impossible?) for Latest to read these.
> 
> If an update is indicated, it typically indicates a new "installer" version available which also updates the bundled Chrome engine etc. So it's always good to download the updated .dmg anyway even if you "already" have the latest version installed.

This was the fix, thanks!  I just downloaded the latest dmg

**Comment by rileysheehan** [comment]
The current suggestion of “just download the latest DMG” works, but it’s not ideal — especially for apps like Obsidian that update frequently via incremental .asar patches.

Would it be possible for Latest to support an alternative version check for certain Electron apps? For example:
- Detect and read the version from app.asar metadata instead of Info.plist when available.
- Allow a per-app override to specify a version source or an online feed (e.g., Obsidian’s release JSON/GitHub API).
- Even a “suppress updates unless installer version changes” toggle for known Electron apps could help reduce noise.

This way, Latest could still notify when an actual installer update is needed (e.g., Chrome engine changes) without incorrectly flagging every incremental app update as requiring a full DMG download.

**Comment by alex-ioma** [comment]
I agree with @rileysheehan - this approach would be highly appreciated. You can always force an `brew upgrade` for the cask, but this creates noise when chrome update is not needed.
